src/containers/Experiment/SamplesTable.js

```diff
@@ -23,20 +21,16 @@ export default class SamplesTable extends React.Component {
     data: []
   };
 
-  handleAddSamplesToDataset = samples => {
-    const { accessionCode: accession_code, addSamplesToDataset } = this.props;
-    addSamplesToDataset([{ accession_code, samples }]);
-  };
-
-  handleRemoveSamplesFromDataset = sampleIds => {
-    const { accessionCode, removeSamplesFromDataset } = this.props;
-    removeSamplesFromDataset(accessionCode, sampleIds);
-  };
+  get totalSamples() {
```

Comment:
Hmm, I don't know I've ever seen `get` in front of a class method declaration? What does this do?

Reply:
It's an [object property](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Functions/get). Basically allows to have a function that can be called as a field. It seemed appropriate for this case... do you think we should use a regular function here?

Reply:
No, that sounds really appropriate here. Didn't know about it, that's super neat ✨